system/Throttle/Throttler.php

```diff
@@ -139,8 +139,7 @@ public function check(string $key, int $capacity, int $seconds, int $cost = 1):
 		$tokenName = $this->prefix . $key;
 
 		// Check to see if the bucket has even been created yet.
-		if (($tokens = $this->cache->get($tokenName)) === null)
-		{
+		if (($tokens = $this->cache->get($tokenName)) === null) {
```

Comment:
This should be reverted to match CodeIgniters coding standard.
{ = should be on it's own line.